### DISCUSSION TITLE
Demo is down

### DISCUSSION THREAD

**Discussion by Mello21century** [filamentphp/filament#3415] [General]
https://demo.filamentphp.com/
Demo is down 503

**Comment by danharrin** [comment]
It's not down, it gets reseeded every half an hour

**Reply by khanakia** [reply]
@danharrin https://filamentphp.com/admin/plugins is giving Server 500 Error. I just tried to register new account from here https://filamentphp.com/plugins > Click Go To Plugin Dashboard page.
